package.json

```diff
@@ -21,10 +21,11 @@
 	"scripts": {
 		"build": "babel --out-dir=dist source",
 		"dev": "babel --out-dir=dist --watch source",
-		"clean": "rm -rf",
-		"setup": "npm run setup:ubuntu",
-		"setup:ubuntu": "npm run clean && npm run build && npm i -g .",
-		"setup:windows":"rmdir /s /q dist && yarn build",
+        "clean:ubuntu": "rm -rf dist",
```

Comment:
Let's add `clean` and default it with the Linux command. So we can use `clean` in a Linux environment as default.

Reply:
ok sure